redux-router/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
+    "@types/react-router": "^2.0.41",
```

Comment:
Doing this would mean that when we make changes to `react-router`, it might break this package and testing wouldn't discover it until we had already published the change to NPM. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/13589#issuecomment-270487446